python/mxnet/_numpy_op_doc.py

```diff
@@ -176,3 +176,229 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
+        Alternative output array in which to place the result.  Must
+        be of the same shape and buffer length as the expected output.
+        See `doc.ufuncs` (Section "Output arguments") for more details.
+
+    keepdims : bool, optional
+        If this is set to True, the axes which are reduced are left
+        in the result as dimensions with size one. With this option,
+        the result will broadcast correctly against the input array.
+
+        If the default value is passed, then `keepdims` will not be
+        passed through to the `amax` method of sub-classes of
+        `ndarray`, however any non-default value will be.  If the
+        sub-class' method does not implement `keepdims` any
+        exceptions will be raised.
+
+    initial : scalar, optional
+        The minimum value of an output element. Must be present to allow
+        computation on empty slice.
+
+        Not supported yet.
+
+    Returns
+    -------
+    amax : ndarray or scalar
+        Maximum of `a`. If `axis` is None, the result is a scalar value.
+        If `axis` is given, the result is an array of dimension
+        ``a.ndim - 1``.
+
+    Examples
+    --------
+    >>> a = np.arange(4).reshape((2,2))
+    >>> a
+    array([[0, 1],
+           [2, 3]])
+    >>> np.amax(a)           # Maximum of the flattened array
+    array(3)
+    >>> np.amax(a, axis=0)   # Maxima along the first axis
+    array([2, 3])
+    >>> np.amax(a, axis=1)   # Maxima along the second axis
+    array([1, 3])
+    >>> np.amax(a, axis=1, keepdims=True)
+    array([[1],
+          [3]])
+
+    Notes
+    -----
+    This function differs to the original `numpy.amax
+    <https://docs.scipy.org/doc/numpy/reference/generated/numpy.amax.html>`_ in
+    the following aspects:
+
+    - The default value type is `float32` instead of `float64` in numpy.
+    - `a` only supports ndarray.
+    - `axis` doe snot support negative value.
+    - `initial` is not supported.
+    """
+    pass
+
+
+def _np_squeeze(a, axis=None):
+    r"""
+    Remove single-dimensional entries from the shape of an array.
+
+    Parameters
+    ----------
+    a : ndarray
+        Input data.
+
+    axis : None or int or tuple of ints, optional
+
+    Returns
+    -------
+    squeezed : ndarray
+        The input array, but with all or a subset of the
+        dimensions of length 1 removed. This is always `a` itself
+        or a view into `a`.
+
+    Examples
+    --------
+    >>> x = np.array([[[0], [1], [2]]])
+    >>> x.shape
+    (1, 3, 1)
+    >>> np.squeeze(x).shape
+    (3,)
+    >>> np.squeeze(x, axis=(2,)).shape
+    (1, 3)
+
+    Notes
+    -----
+    This function differs to the original `numpy.squeeze
+    <https://docs.scipy.org/doc/numpy/reference/generated/numpy.squeeze.html>`_ in
+    the following aspects:
+
+    - The default value type is `float32` instead of `float64` in numpy.
+    - `a` only supports ndarray.
+    """
+    pass
+
+
+def _npi_ones(shape, dtype=None, order='C'):
```

Comment:
Is this an npi operator with a wrapper? If yes then the wrappers' documents will be enough and you do not need to put extra doc in this file.

Reply:
fixed